src/lib/ngsi9/DiscoverContextAvailabilityResponse.cpp

```diff
@@ -101,15 +101,6 @@ std::string DiscoverContextAvailabilityResponse::render(RequestType requestType,
       out += errorCode.render(format, indent + "  ");
   }
 
-#if 0
```

Comment:
Not related with this PR, but spoted and agreed with @kzangeli to remove it. NTC.